### ISSUE TITLE
[QUESTION] Does the example given in the repo work? 

### ISSUE THREAD

**Issue by Aadhinana** [GokiProtocol/walletkit#50]
Running `yarn` and `yarn start` gives this. 
I tried creating a new react project wtih CRA and added `walletkit` to it and still get this error. 
Is this because of `use-solana` being updated or having its code changed? 
Any workarounds this by using an older version? Please help.

![image](https://user-images.githubusercontent.com/22261173/133978715-14aebc78-0006-4f54-b944-79e07bca9ff5.png)

**Comment by Aadhinana** [comment]
Okay so this is weird. 
It works on `http://localhost:3001` but not on `http://localhost:3000`

**Comment by sungoki** [comment]
It should work. It's possible `use-solana` changed-- I'm working on merging this PR https://github.com/GokiProtocol/walletkit/pull/52

**Comment by Aadhinana** [comment]
Yeah looks like it. But it for some reason works on any other port other than 3000. I am doing something wrong?
